features/config/dao/api/src/main/java/org/opennms/features/config/dao/api/ConfigDefinition.java

```diff
@@ -53,11 +90,21 @@ public void setMaxInstances(int maxInstances) {
         this.maxInstances = maxInstances;
     }
 
-    public ConfigItem getSchema() {
-        return schema;
+    public TYPE getType() {
```

Comment:
ConfigItem is our custom data type. It is just a bridge between XSD > ConfigItem > OpenAPI.  Do we really want to store it in database ? The reason I remove that is because we can save processing power do convert it to openapi schema every time we use it.

Reply:
What do we store in the database?

Reply:
the new schema only contain openapi json

Reply:
> ConfigItem is our custom data type. It is just a bridge between XSD > ConfigItem > OpenAPI. Do we really want to store it in database ? The reason I remove that is because we can save processing power do convert it to openapi schema every time we use it.

Ok understand. I'm just wondering if it isn't easier to manipulate java objects instead of Json when changing / upgrading a schema. Maybe we want this conversion optionally.